box-content-sdk/src/main/java/com/box/androidsdk/content/auth/OAuthActivity.java

```diff
@@ -377,7 +378,11 @@ protected void startMakingOAuthAPICall(final String code, final String baseDomai
             return;
         }
         showSpinner();
-        mSession.getAuthInfo().setBaseDomain(baseDomain);
+        if (baseDomain != null) {
+            mSession.getAuthInfo().setBaseDomain(baseDomain);
+        } else {
+            BoxLogUtils.nonFatalE("setting Base Domain", baseDomain, new RuntimeException("base domain being used"));
```

Comment:
This is when we are not using the base domain... please check the comment, or move to the if side